### PR TITLE
Rip out civ config here as well

### PR DESCRIPTION
**Pull request by Maxopoly**
I know that this isn't the cleanest change, but doing this right would require a complete rewrite of CivMenus config handling, the plugin isnt used at all currently and I only want it to compile, because FactoryMod depends on it.

### CODE REVIEW

**Comment by ProgrammerDan** [comment]
Can you doublecheck that the in-line defaults are either present in the default config.yml or superior in the config.yml? Otherwise I'd much prefer that this change faithfully represent those defaults lest they be lost ... otherwise I'm fine with a bit of a rough job on this one.

**Comment by Maxopoly** [comment]
There is no default config at all :(

File: src/vg/civcraft/mc/civmenu/TOSListener.java [hunk 1]
```diff
@@ -28,16 +25,11 @@
...
 	private CivMenu plugin = CivMenu.getInstance();
 	private Map<UUID, Location> locations;
-	private Config config = plugin.GetConfig();
 	
 	public TOSListener() {
 		locations = new ConcurrentHashMap<UUID, Location>();
 	}
 	
-	@CivConfigs({
-		@CivConfig(name = "terms.kickDelay", def = "1200", type = CivConfigType.Int),
```
Comment (ProgrammerDan) [inline src/vg/civcraft/mc/civmenu/TOSListener.java:38]: It's all embedded currently, should be extracted -- I did that for Namelayer and can do that here too I guess, not a big deal tbh.

Comment (Maxopoly) [reply src/vg/civcraft/mc/civmenu/TOSListener.java:38]: Hmm, it's ok, I'll do it myself

**Comment by Maxopoly** [comment]
Should be good enough

**Comment by ProgrammerDan** [comment]
Yeah looks good.

**Pull request merged**